src/main/java/picard/util/BaitDesigner.java

```diff
@@ -489,6 +497,40 @@ protected int doWork() {
             }
         }
 
+        if (AUX_BAITS != null) {
+            final IntervalList auxBaits = IntervalList.fromFile(AUX_BAITS);
+            SequenceUtil.assertSequenceDictionariesEqual(auxBaits.getHeader().getSequenceDictionary(),
+                    baits.getHeader().getSequenceDictionary());
+
+            final ReferenceSequenceFileWalker walker = new ReferenceSequenceFileWalker(REFERENCE_SEQUENCE);
+            ReferenceSequence reference = null;
+            final SAMSequenceDictionary dict = auxBaits.getHeader().getSequenceDictionary();
+            for (final Interval interval : auxBaits.getIntervals()) {
+                final int contigIndex = dict.getSequenceIndex(interval.getContig());
+                if (reference == null || contigIndex != reference.getContigIndex()) {
+                    reference = walker.get(contigIndex);
+                }
+                final Bait bait = new Bait(interval.getContig(),
+                        interval.getStart(),
+                        interval.getEnd(),
+                        interval.isNegativeStrand(),
+                        interval.getName()
+                );
+                bait.addBases(reference, DESIGN_ON_TARGET_STRAND);
+                baits.add(bait);
+            }
+            log.info("Added " + auxBaits.size() + " auxiliary baits");
+        }
+
+        if (AUX_TARGETS != null) {
+            final IntervalList auxTargets = IntervalList.fromFile(AUX_TARGETS);
+            SequenceUtil.assertSequenceDictionariesEqual(auxTargets.getHeader().getSequenceDictionary(),
+                    targets.getHeader().getSequenceDictionary());
+            targets.addall(auxTargets.getIntervals());
+            log.info("Added " + auxTargets.size() + " auxiliary targets");
+
```

Comment:
remove extra space (I know you only added this so that I'll have the pleasure of finding it...)